.specify/scripts/powershell/setup-plan.ps1

```
@@ -0,0 +1,85 @@
+#!/usr/bin/env pwsh
+# Setup implementation plan for a feature
+
+[CmdletBinding()]
+param(
+    [switch]$Json,
+    [switch]$Help,
+    # Capture extra positional arguments to match Bash/Python behavior.
+    [Parameter(ValueFromRemainingArguments = $true)]
+    [string[]]$RemainingArgs
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'RemainingArgs' has been declared but not used.

(PSReviewUnusedParameter)